hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -1655,7 +1694,7 @@ void setIsNamespaceEnabled(final Boolean isNamespaceEnabled) {
    * Getter for abfsCounters from AbfsClient.
    * @return AbfsCounters instance.
    */
-  protected AbfsCounters getAbfsCounters() {
+  public AbfsCounters getAbfsCounters() {
```

Comment:
Is it needs to be protected only for usage in tests?
If so may be we can move the tests in same package (services package inside test)

Reply:
This leads to problems as some AzureBlobFileSystem and AbfsConfiguration methods used in this class are not public and hence cannot be called directly in the services package